configure.ac

```diff
@@ -115,7 +115,7 @@ AC_SUBST([HARDEN_CFLAGS])
 AC_SUBST([HARDEN_LDFLAGS])
 
 # Also check for a few non-hardening-related flags.
-MISC_CXXFLAGS=""
+MISC_CXXFLAGS="-std=c++11"
```

Comment:
`c++ (GCC) 4.1.2 20080704 (Red Hat 4.1.2-48)`

If I revert a4c22e28767832f2cd16820f289b5d4616eeacff, the `-std=c++11` errors go away, but there are still problems in cc336c2908df8f1300b9c35fab381cc343608ebf.  If I also revert that, then this PR builds for me.